libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -115,12 +118,55 @@ export const HealthInsuranceForm: Form = buildForm({
       ],
     }),
     buildSection({
-      id: 'occupation',
-      name: '',
+      id: 'statusAndChildrenSection',
+      name: m.statusAndChildren,
       children: [
-        buildTextField({
-          id: 'occupationText',
-          name: 'text input',
+        buildMultiField({
+          id: 'statusAndChildren',
+          name: m.statusAndChildren,
+          children: [
+            buildRadioField({
+              id: 'status',
+              name: '',
+              description: m.statusDescription,
+              width: 'half',
+              largeButtons: true,
+              options: [
+                {
+                  label: m.statusPensioner,
+                  value: 'pensioner',
+                  tooltip: m.statusPensionerInformation,
+                },
+                {
+                  label: m.statusStudent,
+                  value: 'student',
+                  tooltip: m.statusStudentInformation,
+                },
+                {
+                  label: m.statusOther,
+                  value: 'other',
+                  tooltip: m.statusOtherInformation,
+                },
+              ],
+            }),
+            buildFileUploadField({
```

Comment:
note the file upload field is a bit buggy and might need some improvements before this goes live. Just a heads up